packages/extension-ui/src/Popup/Export.tsx

```diff
@@ -62,12 +62,12 @@ function Export({
       })
       .catch((error: Error) => {
         console.error(error);
-        setError(error.message);
+        setIsProvidedPassWrong(true);
```

Comment:
Not sure if _wrong password_ is the only kind of error that can be caught here (and the other similar spots in this PR). Just rising a concern, no stress about it.